src/components/home/NetworkCard.tsx

```diff
@@ -43,6 +43,13 @@ const NetworkCard: React.FC<{ network: Network }> = ({ network }) => {
             suffix={<LinkOutlined />}
           />
         </Col>
+        <Col span={12}>
```

Comment:
I'm not a big fan of the empty space in the second column. You could get all 3 on one row by setting `span={8}` on all columns and removing the "Nodes" from the titles.